pylutron/__init__.py

```diff
@@ -716,6 +720,28 @@ def is_dimmable(self):
     return self.type != 'NON_DIM' and not self.type.startswith('CCO_')
 
 
+class Shade(Output):
+  """This is the output entity for shades in Lutron universe."""
+  _ACTION_RAISE = 2
+  _ACTION_LOWER = 3
+  _ACTION_STOP = 4
+
+  def start_raise(self):
```

Comment:
do we need the start_ prefix? can we just call this raise/lower?

For the other options for shades if someone wanted them later, I'd assume we'd do something like
    raise_tilt()/lower_tilt()/stop_tilt()
and
    raise_lift()/lower_lift()/stop_lift()

Don't need them now, just talking it out.

Reply:
raise is a reserved word in Python unfortunately...